docs/entity/components.md

```diff
@@ -477,9 +477,10 @@ This feature allows you to upload a single file.
 
 | Property                                                   | Type   | Description                                                | Default Value |
 | ---------------------------------------------------------- | ------ | ---------------------------------------------------------- | ------------- |
-| fileSupportMessage                                          | string | To display message inside file component.                   | -             |
-| supportedFileTypes<span class="required-asterisk">*</span> | array  | A list of the file types that the user can upload.          | -             |
-| maxFileSize                                                | number | To set the maximum file size in KB that a user can upload.  | 500KB         |
+| fileSupportMessage                                         | string | To display message inside file component.                  | -             |
+| supportedFileTypes<span class="required-asterisk">*</span> | array  | A list of the file types that the user can upload.         | -             |
+| maxFileSize                                                | number | To set the maximum file size in KB that a user can upload. | 500KB         |
+| fileNameToDisplay                                          | string | File name used for displaying previous files while edit    | -             |
```

Comment:
yea, i treated it like a general documentation about component but now i read it once again and it looks more like documentation that is focused on mapping globalConfig.json into components. Then yea it does not make sense at all to put it there, it will be just confusing